hexvexed-0.0.1.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="This is a Hexagon based game for haiku"
```

Comment:
In SUMMARY we usually don't have any "This is".
I think "An exagon-based game for Haiku" would be better than "This is a ..."

Can an english native speaker comment on this or suggest somthing better? Thanks!